src/components/GlobalPreferences/CustomLabels/useSelectIdentities.js

```diff
@@ -35,7 +35,7 @@ function useSelectIdentities(identities, filteredIdentities) {
     selected,
   ])
   const handleToggleIdentity = useCallback(
-    address => () =>
+    address =>
```

Comment:
Does it build on https://github.com/aragon/aragon/pull/1072? It seems to include the same changes.

Reply:
Yeah, just rebased to fix it.

Reply:
Ah, that’s much smaller now haha. I usually put “builds on #x” in the PR summary to indicate this.